presto-main/src/main/java/com/facebook/presto/sql/planner/plan/TableScanNode.java

```diff
@@ -73,19 +76,20 @@ public TableScanNode(
             List<Symbol> outputs,
             Map<Symbol, ColumnHandle> assignments)
     {
-        this(id, table, outputs, assignments, TupleDomain.all(), TupleDomain.all(), false);
+        this(id, table, outputs, assignments, TupleDomain.all(), TupleDomain.all(), false, ImmutableMap.of());
     }
 
-    public TableScanNode(
-            PlanNodeId id,
-            TableHandle table,
-            List<Symbol> outputs,
-            Map<Symbol, ColumnHandle> assignments,
-            TupleDomain<ColumnHandle> currentConstraint,
-            TupleDomain<ColumnHandle> enforcedConstraint)
-    {
-        this(id, table, outputs, assignments, currentConstraint, enforcedConstraint, false);
-    }
+//    public TableScanNode(
```

Comment:
@phd3 remove commented out code

Reply:
yes, I just realized a couple things I need to fix. I'd do that and squash commits as well.